codeit/classes/Article.js

```diff
@@ -0,0 +1,13 @@
+class Article {
+    constructor(title, content, writer, likeCount) {
+        this.title = title;
+        this.content = content;
+        this.writer = writer;
+        this.likeCount = likeCount;
```

Comment:
likeCount가 외부에서도 주입이 가능하지만, create로 article을 생성할 수도 있다면, 생성자 내부에서 likeCount를 초기화하는 방향도 고려해보시면 좋을 것 같아요.

```suggestion
        this.likeCount = likeCount ?? 0;
```